state/factory.go

```diff
@@ -440,6 +441,9 @@ func (s *State) Initialize(o common.NamedObject, electionFactory interfaces.IEle
 
 	fmt.Printf("The Instance ID of this node is %s\n", s.Salt.String()[:16])
 
+	s.Blocktime = pubsub.PubFactory.Threaded(10).Publish(pubsub.GetPath(s.GetFactomNodeName(), "blocktime"))
```

Comment:
where was this "blocktime" published prior to your change? was it just missing?

Reply:
Actually CircleCI is failing with this error:
```
panic: failed to publish: publisher already exists at that path (FNode01/blocktime)
```

Reply:
Could it be: https://github.com/WhoSoup/factomd/blob/WhoSoup-bmv_multithreaded/engine/NetworkProcessorNet.go#L75

Reply:
This seems to be collateral damage from #1037 and explains why matt wasn't able to build this one.